ksql-engine/src/test/resources/query-validation-tests/joins.json

```diff
@@ -312,6 +312,27 @@
         {"topic": "INNER_JOIN", "key": 0, "value": {"T_ID": 0, "NAME": "foo", "VALUE": 100, "F1": "zero", "F2": 0}, "timestamp": 10000},
         {"topic": "INNER_JOIN", "key": 0, "value": {"T_ID": 0, "NAME": "bar", "VALUE": 99, "F1": "a", "F2": 10}, "timestamp": 15000}
       ]
+    },
+    {
+      "name": "multi-way join",
```

Comment:
Maybe rename this since `multi-way join` would resemble multiple joins in one query, like 3-way join. How about `join-pipeline`?